dashboard/src/components/NamespaceSelectorMenu.js

```diff
@@ -85,9 +88,10 @@ class NamespaceSelectorMenu extends React.Component {
           organization.environments.map(environment => (
             <Link
               to={`/${organization.name}/${environment.name}`}
+              key={`${organization.name}${environment.name}`}
```

Comment:
The key should be separated by a slash here. Otherwise `ab/c` would have the same key as `a/bc`